.vscode/ruby-lsp/Gemfile.lock

```diff
@@ -0,0 +1,206 @@
+GEM
```

Comment:
Might be nice to `.gitignore` this file so it doesn't need kept up-to-date by Dependabot etc.